### DISCUSSION TITLE
how to recover redefined core operator ?

### DISCUSSION THREAD

**Discussion by kpym** [robpike/ivy#125] [Q&A] [answered]
If I redefine for example `mod` by `op a mod b = 0` how can I recover the original `mod` ?

**Comment by fzipp** [comment]
@kpym It's now possible with https://github.com/robpike/ivy/commit/c7523a413aa10a66ee87e3a5ac5882784e0f4d5d

**Reply by kpym** [reply, marked as answer]
Thanks for the info 👍
